realhf/impl/model/backend/deepspeed.py

```diff
@@ -198,6 +201,18 @@ def train_batch(
         num_micro_batches: Optional[int] = None,
     ):
         if constants.pipe_parallel_world_size() > 1:
+            if num_micro_batches is not None:
+                if num_micro_batches < self.pipe_runner.default_train_mbs:
```

Comment:
The "greater than" should be "less than". @nuzant please check, as well as in the Megatron backend.